src/plugins/intel_cpu/src/emitters/jit_load_store_emitters.hpp

```diff
@@ -15,40 +15,37 @@ using namespace InferenceEngine;
 namespace ov {
 namespace intel_cpu {
 
-struct load_emitter_context : public emitter_context {
-    load_emitter_context() : src_prc_(Precision::FP32), dst_prc_(Precision::FP32), load_num_(8),
-    offset_byte_(0), is_fill_(false), fill_value_("zero") {}
+struct load_emitter_params : public emitter_params {
+    load_emitter_params(Precision src_prc, Precision dst_prc, int load_num, bool is_fill = false, std::string fill_value = "zero"):
+        src_prc_(src_prc), dst_prc_(dst_prc), load_num_(load_num), is_fill_(is_fill), fill_value_(fill_value) {}
 
-    load_emitter_context(Precision src_prc, Precision dst_prc, int load_num, int offset_byte = 0, bool is_fill = false, std::string fill_value = "zero"):
-    src_prc_(src_prc), dst_prc_(dst_prc), load_num_(load_num), offset_byte_(offset_byte), is_fill_(is_fill), fill_value_(fill_value) {}
+    size_t hash() const override;
 
-    int offset_byte_;
-    int load_num_;
     Precision src_prc_;
     Precision dst_prc_;
+    int load_num_;
     bool is_fill_;
     std::string fill_value_;
 };
 
-struct store_emitter_context : public emitter_context {
-    store_emitter_context() : src_prc_(Precision::FP32), dst_prc_(Precision::FP32),
-    store_num_(8), offset_byte_(0) {}
+struct store_emitter_params : public emitter_params {
+    store_emitter_params(Precision src_prc, Precision dst_prc, int store_num):
+        src_prc_(src_prc), dst_prc_(dst_prc), store_num_(store_num) {}
 
-    store_emitter_context(Precision src_prc, Precision dst_prc, int store_num, int offset_byte = 0)
-    : src_prc_(src_prc), dst_prc_(dst_prc), store_num_(store_num), offset_byte_(offset_byte) {}
+    size_t hash() const override;
 
-    int offset_byte_;
     int store_num_;
     Precision src_prc_;
     Precision dst_prc_;
 };
 
 class jit_load_emitter : public jit_emitter {
 public:
-    jit_load_emitter(dnnl::impl::cpu::x64::jit_generator *host, dnnl::impl::cpu::x64::cpu_isa_t host_isa,
-                    InferenceEngine::Precision exec_prc = InferenceEngine::Precision::FP32, emitter_in_out_map in_out_type = emitter_in_out_map::gpr_to_vec);
+    jit_load_emitter(dnnl::impl::cpu::x64::jit_generator *host, dnnl::impl::cpu::x64::cpu_isa_t host_isa, int load_num, Precision src_prc, Precision dst_prc,
```

Comment:
I would propose to move `load_num` after precisions in the argument list to align with `load_emitter_params`